docs/pages/access-controls/access-request-plugins/notification-routing-rules.mdx

```diff
@@ -11,7 +11,7 @@ For example, you may wish to send notifications to different Slack channels
 based on what roles or resources are being accessed, or based on what user
 is making the request.
 
-Currently only the Slack access plugin is supported.
+Currently only the Slack and Mattermost access plugins supported.
```

Comment:
As we're working on enabling them can we rephrase this to avoid changing this every time you add a new plugin? To something like a note-style admonition that says `Currently, only a subset of hosted plugins support notification routing rules. We are working on extending support to the rest of hosted plugins. Keep an eye on Teleport changelog to learn about new plugins being added.` with a link to the changelog page.